<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>about</class>
 <widget class="QDialog" name="about">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>-10</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;版权所有，盗版必究&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="moiveLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>371</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>好的</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>就用</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
